packages/patterns-react/src/patterns/CardSection/__stories__/CardSection.stories.js

```diff
@@ -5,133 +5,12 @@ import { DDS_CARD_SECTION } from '../../../internal/FeatureFlags';
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-    {
-      title: 'Nunc convallis loborti',
-      copy:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-    {
-      title: 'Nunc convallis lbortis',
-      copy:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-  ],
-  imageCards: [
-    {
-      image: {
-        defaultImage: 'https://picsum.photos/id/1003/1056/480',
-        alt: 'cards with image',
-      },
-      title: 'Topic',
-      copy: 'Natural language processing.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-    {
-      image: {
-        defaultImage: 'https://picsum.photos/id/1018/1056/480',
-        alt: 'cards with image',
-      },
-      title: 'Blog',
-      copy: 'Natural language processing.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-    {
-      image: {
-        defaultImage: 'https://picsum.photos/id/1076/1056/480',
-        alt: 'cards with image',
-      },
-      title: 'Topic',
-      copy: 'Natural language processing.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-    {
-      image: {
-        defaultImage: 'https://picsum.photos/id/102/1056/480',
-        alt: 'cards with image',
-      },
-      title: 'Blog',
-      copy: 'Serving society ethically in the age of Artificial Intelligence.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-    {
-      image: {
-        defaultImage: 'https://picsum.photos/id/1032/1056/480',
-        alt: 'cards with image',
-      },
-      title: 'Topic',
-      copy: 'Serving society ethically in the age of Artificial Intelligence.',
-      link: {
-        href: 'https://www.example.com',
-        text: 'Learn more',
-        target: '_self',
-      },
-    },
-  ],
-};
-
 if (DDS_CARD_SECTION) {
-  storiesOf('CardSection', module)
+  storiesOf('Sub-Patterns|CardSection', module)
```

Comment:
This is the only one that I think is incorrect at the moment. I think it belongs in `Section`, but I understand if we want to keep this the way it is until we bring in other sub-patterns so we have this placeholder as an example.

Reply:
Groovy yeah, I will fix this in the other PR where I re-org everything.